src/etc/inc/plugins.inc.d/vxlan.inc

```diff
@@ -110,14 +112,29 @@ function vxlan_configure_do($verbose = false, $device = null)
             $current_settings['vxlanid'] = $interfaces_details[$device_name]['vxlan']['vni'];
             $current_settings['vxlanlocal'] = explode(":", $interfaces_details[$device_name]['vxlan']['local'])[0];
             $current_settings['vxlanremote'] = explode(":", $interfaces_details[$device_name]['vxlan']['remote'])[0];
-            if (!empty($interfaces_details[$device_name]['vxlan']['group'])) {
-                $current_settings['vxlangroup'] = explode(":", $interfaces_details[$device_name]['vxlan']['group'])[0];
+            $current_settings['vxlanlocalport'] = explode(":", $interfaces_details[$device_name]['vxlan']['local'])[1];
```

Comment:
@Monviech at a first glance the assessment in the PR comment looks correct, but we can simplify this quite a bit in my opinion. Looking at the ticket we're trying to set "local" and "remote" so "group" would be good as it was, although we can simplify it in the same loop.

Something like this might save some code:

```
foreach (['local', 'remote', 'group'] as $target) {
    if (!empty($current_settings[$interfaces_details[$device_name]['vxlan'][$target]]) {
        $tmp = explode(':', $interfaces_details[$device_name]['vxlan'][$target]]);
        $current_settings['vxlan'.$target] = $tmp[0];
        $current_settings['vxlan'.$target.'port'] = $tmp[1];
    }
}
```